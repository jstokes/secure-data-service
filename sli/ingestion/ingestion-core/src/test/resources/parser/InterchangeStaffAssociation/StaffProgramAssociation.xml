<?xml version="1.0" encoding="UTF-8"?>
<InterchangeStaffAssociation xmlns:sli="http://slc-sli/ed-org/0.1" xmlns="http://ed-fi.org/0100"
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <StaffProgramAssociation>
    <StaffReference>
      <StaffIdentity>
        <StaffUniqueStateId>Qm5qi.fpz1l.02A</StaffUniqueStateId>
      </StaffIdentity>
    </StaffReference>
    <ProgramReference>
      <ProgramIdentity>
        <ProgramId>LDDUdY</ProgramId>
      </ProgramIdentity>
    </ProgramReference>
    <BeginDate>2013-08-14</BeginDate>
    <EndDate>2014-07-21</EndDate>
    <StudentRecordAccess>false</StudentRecordAccess>
  </StaffProgramAssociation>
</InterchangeStaffAssociation>
