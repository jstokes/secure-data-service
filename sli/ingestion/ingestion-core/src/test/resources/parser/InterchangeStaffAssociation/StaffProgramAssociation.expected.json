{
  "StaffReference":{
    "StaffIdentity":{
      "StaffUniqueStateId":{"_value":"Qm5qi.fpz1l.02A"}
    }
  },
  "ProgramReference":{
    "ProgramIdentity":{
      "ProgramId":{"_value":"LDDUdY"}
    }
  },
  "BeginDate":{"_value":"2013-08-14"},
  "EndDate":{"_value":"2014-07-21"},
  "StudentRecordAccess":false
}